src/cli/ui/shared.py

```
@@ -0,0 +1,117 @@
+
+from prompt_toolkit.formatted_text import HTML, FormattedText
+from prompt_toolkit.layout.containers import Window
+from prompt_toolkit.layout.controls import FormattedTextControl
+from prompt_toolkit.styles import Style
+
+from src.cli.theme import theme
+from src.core.config import ApprovalMode
+from src.utils.version import get_version
+
+
+def get_prompt_color(context, bash_mode: bool = False) -> str:
```

Ruff (0.14.6)

Boolean-typed positional argument in function definition

(FBT001)